### ISSUE TITLE
Jorden_net Only predicting no wires 

### ISSUE THREAD

**Issue by NotJoeMartinez** [NotJoeMartinez/APDL_Quality_Control#22]
This has something to do with the testing, unlikely the model
![image](https://user-images.githubusercontent.com/39905973/120410235-3160f000-c318-11eb-9aa0-a8060706788b.png)

**Comment by NotJoeMartinez** [comment]
This might be caused by overrepresentation in the training dataset. 
![image](https://user-images.githubusercontent.com/39905973/120506312-69ede180-c38b-11eb-8cdb-56f35674353f.png)

**Comment by NotJoeMartinez** [comment]
closed with  52495ca

in `test_model.py`
```python
# normalized_image_array = (image_array.astype(np.float32) / 127.0) - 1
normalized_image_array = image_array.astype(np.float32)
```
